src/components/InputData/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react"
+import TextField from "@mui/material/TextField"
+import { blue } from "@mui/material/colors"
+import { InputAdornment } from "@mui/material"
+
+type InputProps = {
+  data?: string
+}
+
+function InputComponent({ data }: InputProps) {
+  const [value, setValue] = useState("")
+  const [copied, setCopied] = useState(false)
+  const [clicked, setClicked] = useState(false)
+
+  const handleCopyClick = () => {
+    navigator.clipboard.writeText(value)
+    setCopied(true)
+    setClicked(true)
+  }
+
+  const handleInputFocus = () => {
+    setClicked(false)
+  }
+
+  return (
+    <div>
+      <TextField
+        type="text"
+        value={data}
+        inputMode="text"
+        onChange={(event) => setValue(event.target.value)}
+        InputProps={{
+          style: {
+            backgroundColor: clicked ? blue[50] : "transparent",
+          },
+          onFocus: handleInputFocus,
+          endAdornment: (
+            <InputAdornment position="end">
+              <img
+                src="src\assets\CopyInput.png"
```

Comment:
You can import the image and use it like this:
```suggestion
import CopyInput from "../../assets/CopyInput.png"
                src={CopyInput}
```

Reply:
Ok, did the requested changes